class_exercises/exerciseproblemsolving.js

```diff
@@ -0,0 +1,36 @@
+//we need an array of numbers
+//we need to reverse the array
+//if/else statement that checks the number is greater or less than 10
+//this should be inside of a for loop
+
+
+
+
+var numbers = [0,20,56,678];
+
+numbers.reverse();
+
+for (var i = 0; i< numbers.length; i++){
+		
+		if (numbers[i] < 10){
+		console.log('less is more');
```

Comment:
this code is valid, but the tabbing is inconsistent. It's good to get in the habit of using consistent tabbing now, because it will be expected in a professional environment. :D